### DISCUSSION TITLE
Cross compile VVdeC and VVenC

### DISCUSSION THREAD

**Discussion by ZenKiyoshi** [fraunhoferhhi/vvdec#106] [Q&A] [answered]
How to cross compile VVdeC and VVenC?

Ubuntu host, Windows executable file.

**Comment by K-os** [comment, marked as answer]
Use mingw  as a compiler (e.g. Ubuntu package g++-mingw-w64-x86-64) and provide a corresponding toolchain file to the build. Either you invoke CMake direcly and pass the variable `-DCMAKE_TOOLCHAIN_FILE=<your_toolchain_file>` or you use the Makefile wrapper and call `make release toolchainfile=<your_toolchain_file>`. For Ubuntu we even provide toolchain files in the directory `cmake/toolchains/`.

Compiling should work in principle because we build it regularly within our CI-pipeline, but we don't test the resulting binaries.
